### PR TITLE
Add a helper method to obtain the namespace of a pod.

### LINKED ISSUE #1313: Get Namespace When Running In Pod
By default there is a file called `namespace` located at `/var/run/secrets/kubernetes.io/serviceaccount` when running in a pod.  Is there a way of reading that files value in the client library?  I poked around but couldnt find anything.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes #1313

### CODE REVIEW

File: util/src/main/java/io/kubernetes/client/util/ClientBuilder.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
   private byte[] caCertBytes = null;
   private boolean verifyingSsl = true;
   private Authentication authentication;
+  private String namespace;
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/util/ClientBuilder.java:51]: is the new field in use? @brendandburns

Comment (brendandburns) [reply util/src/main/java/io/kubernetes/client/util/ClientBuilder.java]: oops, removed.

**Review by yue9944882 — COMMENTED** [review]
/lgtm

**Comment by ryanjbaxter** [comment]
Thanks!

**Pull request merged**